plugins/binanceExchange_ws.go

```diff
@@ -209,6 +261,33 @@ func getPrecision(floatStr string) int8 {
 	return int8(len(strs[1]))
 }
 
+//subscribeStream and wait for the first event
+func (beWs *binanceExchangeWs) subscribeStream(symbol, format string, subscribe Subscriber, state *mapEvents) (mapData, error) {
+
+	stream, err := subscribe(symbol, state)
+
+	if err != nil {
+		return mapData{}, fmt.Errorf("error when subscribing for %s: %s", symbol, err)
+	}
+
+	//Store stream
+	beWs.streamLock.Lock()
+	beWs.streams[fmt.Sprintf(format, symbol)] = stream
+	beWs.streamLock.Unlock()
+
+	//Wait for binance to send events
+	time.Sleep(timeToWaitForFirstEvent)
+
+	data, isStream := beWs.events.SymbolStats.Get(symbol)
+
+	//We couldn't subscribe for this pair
+	if !isStream {
+		return mapData{}, fmt.Errorf("error while fetching ticker price for trading pair %s", symbol)
```

Comment:
Got this error in the test. This says ticker price but it should say orderbook. we cannot hardcode "ticker" over here. I think the error is above on line 281 where we use `beWs.events.SymbolStats` and instead we should use `state`.

Can you check the refactored functions to ensure we've replaced `beWs.events.SymbolStats` where applicable?

```bash
2021/07/18 15:46:10 Ticker for XLMBTC is 267 milliseconds old!
ticker price: bid=0.00000731, ask=0.00000732, last=0.00000732
--- FAIL: Test_binanceExchangeWs_GetOrderBook (1.59s)
    binanceExchange_ws_test.go:63: 
        	Error Trace:	binanceExchange_ws_test.go:63
        	Error:      	Received unexpected error:
        	            	error while fetching ticker price for trading pair XLMBTC
        	Test:       	Test_binanceExchangeWs_GetOrderBook
```